uint8 POSITION_CONTROL=0
uint8 VELOCITY_CONTROL=1
uint8 CLICK_CONTROL=2
uint8 TYPING=3
uint8 mode
---
bool success
